draft-ietf-quic-manageability.md

```diff
@@ -823,9 +824,17 @@ techniques and how QUIC's design impacts them.
 
 Limited RTT measurement is possible by passive observation of QUIC traffic;
 see {{sec-rtt}}. No passive measurement of loss is possible with the present
-wire image. Extremely limited observation of upstream congestion may be
+wire image. Limited observation of upstream congestion may be
 possible via the observation of CE markings on ECN-enabled QUIC traffic.
 
+On-path devices can also make measurements of RTT, loss and other
+performance metrics when information is carried in an additional network-layer
+packet header (Section 6 of
+{{?I-D.ietf-tsvwg-transport-encrypt}} describes use of operations,
+administration and management (OAM) information).
+Using network-layer approaches also has the potential that common observation
+and analysis tools can be consistently used by multiple transport protocols.
```

Comment:
```suggestion
and analysis tools can be consistently used by multiple transport protocols,
however, these techniques are often limited to measurements within one or
multiple cooperating domains.
```
Proposing this change as it otherwise sounds that we don't need any in-band e2e measurement support which I don't think is true or intended. However, let me know if what I wrote is actually correct...?

Reply:
I think this is a reasonable addition, but I'd prefer a separate sentence starting with "However, "